app/subscriber/src/features/content/view-content/styled/ViewContent.tsx

```diff
@@ -10,7 +10,7 @@ export const ViewContent = styled.div`
     margin-right: -1.25em;
     padding-left: 1.25em;
     padding-right: 1.25em;
-    font-size: 0.8rem;
+    font-size: 0.9rem;
```

Comment:
should this be `em` ?

Reply:
Not sure, `rem` is relative to root font size, so it'd be more consistent sizing. Whereas `em` is relative to parent font size. I'd think `rem` should be used more generally